openquake/hazardlib/gsim/stewart_2016_vh.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2012-2021 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`StewartEtAl2016VH`,
+               :class:`StewartEtAl2016RegCHNVH`,
+               :class:`StewartEtAl2016RegJPNVH`,
+               :class:`StewartEtAl2016NoSOFVH`,
+               :class:`StewartEtAl2016RegCHNNoSOFVH`,
+               :class:`StewartEtAl2016RegJPNNoSOFVH`,
+"""
+
+import numpy as np
+
+import openquake.hazardlib.gsim.boore_2014 as BSSA
+import openquake.hazardlib.gsim.bozorgnia_campbell_2016_vh as BC15b
```

Comment:
BSSA and BC15b are too cryptic, why don't you just use the module names?

```python
from openquake.hazardlib.gsim import bozorgnia_campbell_2016_vh
from openquake.hazardlib.gsim import boore_2014
```

Reply:
I followed the acronyms of the GMPEs and as found in the [openquake/hazardlib/tests/gsim/data](https://github.com/gem/oq-engine/tree/master/openquake/hazardlib/tests/gsim/data) directory to avoid the long module names. Although, it might be fine in this case since `bozorgnia_campbell_2016_vh` is only used thrice and still spanning one line each.